src/Maestro/Maestro.DataProviders/MaestroBarClient.cs

```diff
@@ -447,7 +467,11 @@ public async Task<BuildTime> GetBuildTimeAsync(int defaultChannelId, int days)
                 };
             }
 
-            MultiProjectKustoQuery queries = SharedKustoQueries.CreateBuildTimesQueries(defaultChannel.Repository, defaultChannel.Branch, days);
+            MultiProjectKustoQuery queries = SharedKustoQueries.CreateBuildTimesQueries(
+                defaultChannel.Repository,
+                defaultChannel.Branch,
+                days,
+                defaultChannel.BuildDefinitionId);
```

Comment:
Aren't the definition ids for PRs and definition ids for official builds different? I'm trying to page this code back into my brain, but I'm pretty sure this is trying to get the time for the official build time in addition to the PR time (because dependency flow "worst case" is PR time+official build time, and "best case" is just official build time).

Reply:
You're right. I changed it so that only internal builds are filtered. Also I changed `BuildTimeController` so that it reuses the same logic. I will test it some more on Monday to make sure all scenarios work ok.